### DISCUSSION TITLE
Calendar

### DISCUSSION THREAD

**Discussion by Rychillie** [nandorojo/solito#65] [Q&A] [answered]
Hey @nandorojo I was looking at the [Beatgig](https://beatgig.com) website and I saw that you use a calendar to organize your partner's agenda, could you share which library you used or the way you made it to have this format?

I plan to implement a calendar system like this in my personal personal organization App!

**Comment by nandorojo** [comment, marked as answer]
full calendar for the large one, react-native-calendar for the small one
